docs/how-ubuntu-is-made/concepts/version-strings.md

```diff
@@ -158,15 +167,15 @@ package used so far.
 
 List of this and further related examples:
 
-| Previous version                        | No-change rebuild in devel   |
-| --------------------------------------- | ---------------------------- |
-| `2.0-2`                                 | `2.0-2build1`                |
-| `2.0-2ubuntu2`                          | `2.0-2ubuntu3`               |
-| `2.0-2build1`                           | `2.0-2build2`                |
-| `2.0ubuntu0` (native in Ubuntu)         | `3.0ubuntu0` or `2.1ubuntu0` |
-| `2.0-0ubuntu1` (version only in Ubuntu) | `2.0-0ubuntu2`               |
```

Comment:
This is a matter of converging old to new style.
It is right to add the new lines, but I think you need to keep the old lines and update them on the right how they are expected to convert to the new. What shall become of 2.0ubuntu0 or 2.0-0ubuntu1` that already exist due to $history?